.github/workflows/live-service-testing.yml

```diff
@@ -58,7 +61,8 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           s3-gin-bucket: ${{ secrets.S3_GIN_BUCKET }}
-
+      - name: Run subset of tests that use EMBER live services
```

Comment:
@CodyCBakerPhD I did change this but I am still getting errors. Is this what you meant?